guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -1,21 +1,21 @@
- Adam Růžička <adamruzicka@users.noreply.github.com>
```

Comment:
I assume this file was created automatically. Maybe for future releases, we should instead rework this script?

You change is fine for me though!

Reply:
I don't know how it was created, but https://github.com/theforeman/foreman-documentation/issues/1767 is open to unify it.